cost/azure/savings_plan/recommendations/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## v2.0
```

Comment:
V1.0 for initial releases in my opinion for now. 

Eventually we should consider switching strictly to using Semantic Versioning (semver.org has the spec) and initial releases would be v0.1.0. Have discussed with @taterdog590 as well and just something we need to agree on and add to PR tests

Reply:
Updated to reflect v1.0 in both changelog and policy template

Reply:
Historically, v1.0 is reserved for policies that were before Policy Credentials were released instead of Cloud Management Credentials.  v2.0 used Credentials.  We should follow that example.  This policy should be v2.0